python/aswfdocker/data/versions.yaml

```diff
@@ -16,6 +16,8 @@ versions:
       ASWF_CONAN_PYTHON_VERSION: "3.9.11"
       ASWF_NINJA_VERSION: "1.10.1"
       ASWF_SONAR_VERSION: "4.6.2.2472"
+      ASWF_BASEOS_VERSION: "centos7"
+      ASWF_BASEOS_REPOSITORY: "nvidia/cudagl"
```

Comment:
Would this be `ASWF_BASEOS_IMAGENAME`?

Reply:
Do you mean renaming ASWF_BASEOS_REPOSITORY to ASWF_BASEOS_IMAGENAME? I agree that's better.

Reply:
These are now:

```
      ASWF_BASEOS_DISTRO: "rockylinux8"
      ASWF_BASEOS_IMAGE: "nvidia/cuda"
```